backend/src/main/java/codezap/member/dto/LoginRequest.java

```diff
@@ -0,0 +1,27 @@
+package codezap.member.dto;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public record LoginRequest(
+        @Schema(
+                description = "이메일",
+                example = "code@zap.com",
+                pattern = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$\n"
+        )
+        @NotEmpty(message = "이메일이 입력되지 않았습니다.")
+        @Size(max = 255, message = "이메일은 255자 이하로 입력해주세요.")
+        String email,
```

Comment:
```suggestion
        @Schema(
                description = "이메일",
                example = "code@zap.com",
                pattern = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$\n"
        )
        @NotEmpty(message = "이메일이 입력되지 않았습니다.")
        @Size(max = 255, message = "이메일은 255자 이하로 입력해주세요.")
        @Pattern(regexp = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$\n")
        //@Email(regexp = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$\n")
        String email,
```

`@Schema` 에서 정규 표현식을 명시해 두었으니만큼, `@Pattern` 또는 `@Email` 을 검증에도 사용해 보는 것도 좋을 것 같아요.